RangeShiftR/src/RScore/Genome.cpp

```diff
@@ -124,12 +124,14 @@ void Chromosome::inherit(const Chromosome* parentChr, const short posn, const sh
 	for (int i = 0; i < nloc; i++) {
 		if (diploid) {
 			pLoci[i].allele[posn] = parentChr->pLoci[i].allele[ix];
+		    if (probcross >1) Rcpp::Rcout << "Crossover probability: " << probcross << std::endl;
```

Comment:
I probably can get rid of this line again. (but it will also probably be deleted anyways when adding the new genetics.

Reply:
Done